packages/cli/generators/extension/templates/src/component.ts.ejs

```diff
@@ -1,9 +1,13 @@
 import {Component, ProviderMap} from '@loopback/core';
+import {<%= project.name.charAt(0).toUpperCase() + project.name.slice(1) %>Bindings} from './keys'
```

Comment:
We should not perform case manipulation in the template. Please move this code to the generator and make sure we use the same logic for all places (templates) that need to acces `{ProjectName}Bindings`.

```suggestion
import {<%= project.bindingsNamespace %>} from './keys'
```

See how we `lb4 app` is building `appClassWithMixins` string for inspiration:

https://github.com/strongloop/loopback-next/blob/4fbe6afb19f689b3a27aa4992377e3993ab62973/packages/cli/generators/app/index.js#L118-L134